packages/graphql/src/translate/queryAST/ast/filters/authorization-filters/JWTFilter.ts

```diff
@@ -29,35 +29,58 @@ export class JWTFilter extends Filter {
     protected operator: FilterOperator;
     protected JWTClaim: Cypher.Property;
     protected comparisonValue: unknown;
+    protected isNot: boolean;
 
     constructor({
         operator,
         JWTClaim,
         comparisonValue,
+        isNot,
     }: {
         operator: FilterOperator;
         JWTClaim: Cypher.Property;
         comparisonValue: unknown;
+        isNot: boolean;
     }) {
         super();
         this.operator = operator;
         this.JWTClaim = JWTClaim;
         this.comparisonValue = comparisonValue;
+        this.isNot = isNot;
     }
 
     public getChildren(): QueryASTNode[] {
         return [];
     }
 
     public getPredicate(_context: QueryASTContext): Predicate | undefined {
-        return createComparisonOperation({
+        if (this.comparisonValue === null) {
+            return this.getNullPredicate(this.JWTClaim);
+        }
+
+        const operation = createComparisonOperation({
             operator: this.operator,
             property: this.JWTClaim,
             param: new Cypher.Param(this.comparisonValue),
         });
+
+        return this.wrapInNotIfNeeded(operation);
     }
 
     public print(): string {
         return `${super.print()} <${this.operator} ${this.comparisonValue}>`;
     }
+
+    private getNullPredicate(propertyRef: Cypher.Property): Cypher.Predicate {
+        if (this.isNot) {
+            return Cypher.isNotNull(propertyRef);
+        } else {
+            return Cypher.isNull(propertyRef);
+        }
+    }
+
+    private wrapInNotIfNeeded(predicate: Cypher.Predicate): Cypher.Predicate {
+        if (this.isNot) return Cypher.not(predicate);
+        else return predicate;
```

Comment:
```suggestion
        if (this.isNot) {
            return Cypher.not(predicate);
        } else {
            return predicate;
        }
```

I think we should try to keep a consistent style here.

Reply:
You can have a chat to the lovely @angrykoala on this coding standard. Whilst I completely agree with you, this function was copied from elsewhere in the translation layer.

But I've fixed it in this instance.